tests/conftest.py

```diff
@@ -86,3 +90,29 @@ def boto_session():
         aws_session_token="testing",
         region_name='us-east-1',
     )
+
+
+@pytest.fixture
+def test_client(boto_session):
+    private_key = "secret123file"
+    with tempfile.NamedTemporaryFile() as fp:
+        fp.write(private_key.encode('utf-8'))
+        fp.flush()
+        private_key_path = fp.name
+        issuer = ("atomeka_test",)
+        kid = ("atomekadefault_test",)
```

Comment:
Did you mean to set the `issuer` and `kid` to tuples instead of strings?

Reply:
ahh I think I copied this code from another unit test. Tuples is definitely unnecessary for this! Will change to use strings.